src/query/service/src/interpreters/interpreter_select.rs

```diff
@@ -259,7 +259,7 @@ impl Interpreter for SelectInterpreter {
             .format(self.metadata.clone(), Default::default())?
             .format_pretty()?;
         info!(
-            "Query id: {}, query plan: \n{}",
+            "Query id: {}, query plan: {}",
             self.ctx.get_id(),
             query_plan
```

Comment:
query_plan also contains '\n'

Reply:
Umm, we will fix it in the future. Will merge this PR and create a new release for prod.